benchmark/bert.cpp

```diff
@@ -72,6 +72,8 @@ static void setupDivMaxSoftmaxDropoutForward(Fusion* fusion, DataType dtype) {
 }
 
 static void setupDivMaxSoftmaxDropoutBackward(Fusion* fusion, DataType dtype) {
+  FusionGuard fg(fusion);
+
   TensorView* tv0 = makeContigTensor(4, dtype);
   // Strangely tv1 isn't used anywhere, need to come back to that...
   TensorView* tv1 = makeContigTensor(4, dtype);
```

Comment:
see we have `y = model(x)`. When we do backward to calculate the gradient of x we usually have `x_gradient = backward_model(x,z) `where `z` is the upstream gradient. Here `tv1` seems to be `y` which is not used. Just a guess.